README.md

```diff
@@ -1,3 +1,18 @@
+# OMI Python Script Provider
+
+## How to build
+
+### Dependencies to build OMI Python Script Provider:
+- Install gcc and g++
+- Install python and python-devel

```

Comment:
It might be better to provide more details on the versions that need to be installed.